src/lib.rs

```diff
@@ -1,28 +1,123 @@
 use std::fmt::Write;
 
-#[derive(PartialEq, Eq)]
-pub struct Roff {
+/// Title line for a manpage.
+pub struct Title {
```

Comment:
We should at least implement `Debug` and `Clone` on everything

Reply:
Good point!